cmd/create/cluster/cmd.go

```diff
@@ -2368,10 +2368,8 @@ func handleOidcConfigOptions(r *rosa.Runtime, cmd *cobra.Command, isSTS bool, is
 		}
 	}
 	if oidcConfigId == "" {
-		if !isHostedCP {
-			return nil
```

Comment:
The only thing is that I feel like this message should show only for this one

If the cluster is a hosted CP cluster and the user specified args.classicOidcConfig it doesn't mean that an oidc configuration was not found but that the check was skipped, so it means the customer doesn't even want to check if there are any.